build.gradle

```diff
@@ -1,22 +1,18 @@
 plugins {
-  id 'org.owasp.dependencycheck' version '8.0.2'
+  id 'org.owasp.dependencycheck' version '7.3.0'
```

Comment:
This should be set to the latest version, which I believe is currently 8.2.1

Reply:
Changed to the latest version